llms.txt

```diff
@@ -145,6 +145,7 @@ Doc-Page: https://docs.polkadot.com/tutorials/polkadot-sdk/system-chains/
 Doc-Page: https://docs.polkadot.com/tutorials/polkadot-sdk/testing/fork-live-chains/
 Doc-Page: https://docs.polkadot.com/tutorials/polkadot-sdk/testing/
 Doc-Page: https://docs.polkadot.com/tutorials/polkadot-sdk/testing/spawn-basic-chain/
+Doc-Page: https://docs.polkadot.com/venv/lib/python3.13/site-packages/idna-3.10.dist-info/LICENSE/
```

Comment:
```suggestion
```

Reply:
GitHub does not allow us to apply these suggestions. I'll merge this to a feature branch and manually fix it